docs/site/pdf_generation.md

```diff
@@ -12,7 +13,8 @@ Selenium is documented below.
    and may be deprecated in the future.
 
 ## Installation
-Install [Selenium](https://www.selenium.dev/) in the same Python virtual environment as the site, and install the
+
+Install [Selenium](https://www.selenium.dev/) in the same Python virtual environment as the site, and install
```

Comment:
Maybe put the "install ChromeDriver" part in the second sentence all together? Sounds pretty weird now.

Reply:
I rewrote this sentence.